src/CliMenu.php

```diff
@@ -640,6 +655,50 @@ public function setStyle(MenuStyle $style) : void
         $this->style = $style;
     }
 
+    public function getCheckboxStyle() : CheckboxStyle
+    {
+        return $this->checkboxStyle;
+    }
+
+    /**
+     * Use as
+     *
+        ->checkboxStyle(function (CheckboxStyle $style) {
+            $style->setMarkerOff('- ');
+        })
+     *
+     * @param callable $itemCallable
+     * @return $this
+     */
+    public function checkboxStyle(callable $itemCallable) : self
```

Comment:
maybe `modifyCheckboxStyle(callable $itemCallable)` ?

Reply:
I'm also not sure about this method - why is this needed, it's a duplicate of the `CliMenuBuilder`. Isn't `get` and `set` enough here?

Reply:
Returning `CheckboxStyle()` breaks the fluent interface:

```
$menu = (new CliMenuBuilder)
    ->setTitle('Select a Language')
    ->getCheckboxStyle()
    ->setMarkerOn('[○] ')
    // Can't continue here because setMarkerOn() returns CheckboxStyle
    ->addCheckboxItem('PHP', $itemCallable)
    ->build();
```

vs making it a callback:

```
$menu = (new CliMenuBuilder)
    ->setTitle('Select a Language')
    ->checkboxStyle(function (CheckboxStyle $style) {
        $style->setMarkerOff('[○] ')
            ->setMarkerOn('[●] ');
    })
    // The above returns $this referencing $menu, allows fluent interface to continue
    ->addCheckboxItem('PHP', $itemCallable)
    ->build();
``

Reply:
I just mean on `CliMenu` not on `CliMenuBuilder`. I don't think we need the fluent interface on `CliMenu`.

Reply:
Gotcha, that makes sense.